### PR TITLE
update to repos standards

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Comment by SimonCropp** [comment]
@Particular/nservicebus-maintainers this needs to be merged since the inconsistency of the repo standards in the source only packages is breaking the downstreams

**Comment by danielmarbach** [comment]
@SimonCropp thanks will review and merge

File: src/NServiceBus.Core/Routing/MessagingBestPractices/BestPracticesOptionExtensions.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public static void DoNotEnforceBestPractices(this ExtendableOptions options)
         /// Returns whether <see cref="DoNotEnforceBestPractices(NServiceBus.Extensibility.ExtendableOptions)" /> has ben called or
         /// not.
         /// </summary>
-        /// <returns><c>true</c> if best practice enforcment has ben disabled, <c>false</c> otherwhise.</returns>
+        /// <returns><c>true</c> if best practice enforcement has ben disabled, <c>false</c> otherwise.</returns>
```
Comment (danielmarbach) [inline src/NServiceBus.Core/Routing/MessagingBestPractices/BestPracticesOptionExtensions.cs]: Nice catch

File: src/NServiceBus.Core/Routing/FileBasedDynamicRouting/FileRoutingTableSettings.cs [hunk 2]
```diff
@@ -31,7 +31,7 @@ public FileRoutingTableSettings RefreshInterval(TimeSpan refreshInterval)
         /// <summary>
         /// Specifies the maximum number of attempts to load contents of a file before logging an error.
         /// </summary>
-        /// <param name="maxLoadAttempts">Max load attepts.</param>
+        /// <param name="maxLoadAttempts">Max load attempts.</param>
```
Comment (danielmarbach) [inline src/NServiceBus.Core/Routing/FileBasedDynamicRouting/FileRoutingTableSettings.cs]: Nice catch

File: src/NServiceBus.Core/Recoverability/Faults/ErrorQueueSettings.cs [hunk 3]
```diff
@@ -27,10 +27,10 @@ public static string GetConfiguredErrorQueue(ReadOnlySettings settings)
                 }
 
                 throw new Exception(
-                    @"'MessageForwardingInCaseOfFaultConfig' configuration section is found but 'ErrorQueue' value is empty. 
-Ttake on of the following actions: 
+                    @"'MessageForwardingInCaseOfFaultConfig' configuration section is found but 'ErrorQueue' value is empty.
```
Comment (danielmarbach) [inline src/NServiceBus.Core/Recoverability/Faults/ErrorQueueSettings.cs]: Nice catch

File: src/NServiceBus.Core/obsoletes.cs [hunk 4]
```diff
@@ -1073,7 +1073,7 @@ public TransactionSettings(bool isTransactional, TimeSpan transactionTimeout, Is
         public bool DoNotWrapHandlersExecutionInATransactionScope { get; set; }
 
         [ObsoleteEx(
-            Message = "SuppressDistributedTransactions is no longer used here. Uuse `context.Settings.GetRequiredTransactionModeForReceives() != Transactions.TransactionScope` instead.",
+            Message = "SuppressDistributedTransactions is no longer used here. Use `context.Settings.GetRequiredTransactionModeForReceives() != Transactions.TransactionScope` instead.",
```
Comment (danielmarbach) [inline src/NServiceBus.Core/obsoletes.cs]: Nice catch

**Pull request merged**